src/open_inwoner/scss/views/App.scss

```diff
@@ -253,6 +253,26 @@
   --form-width: 500px;
 }
 
+// Custom fonts with fallback
+@font-face {
+  font-family: 'Body';
+  src: local('Body'),
+    url('/media/custom_fonts/text_body_font.otf') format('opentype'),
```

Comment:
This won't work for TTF and WOFF files. Or: ⚠️ we would have to explicitly tell the user in the admin to only use OTF files. It IS possible to convert TTF files to OTF online but for some fonts this could lead to conversion problems. 
Fontsquirrel turns fonts into WOFF2 by default, but with Fontforge, i think, you can use a Python/PIP script to turn them into OTF and any other type for multiple compatibility: https://github.com/fontforge/fontforge

You can download test font files from Fontsquirrel,
Like for example: https://www.fontsquirrel.com/fonts/work-sans where you'll see TTF is common,
Different font-file types would need a different specification in CSS, it is possible to use font-face like this: 
```url('/media/custom_fonts/text_body_font.otf') format('opentype'),
url('/media/custom_fonts/text_body_font.woff') format('woff'),
url('/media/custom_fonts/text_body_font.eot')
      format('embedded-opentype'),
url('/media/custom_fonts/text_body_font.ttf') format('truetype'),
url('/media/custom_fonts/text_body_font.svg') format('svg');```

Reply:
Yeah I thought we would only support OTF (you're right this should be made explicit in the help text if we take this route). The issue with supporting alternatives is that this will lead to a bunch of `404`s with browsers failing to find the specified fonts. Let's discuss this in the office.